exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -0,0 +1,158 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class TournamentTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void justTheHeaderIfNoInput() {
+        Tournament tournament = new Tournament();
```

Comment:
This line is repeated in all the tests to it can be put in a [`@Before` block](https://www.mkyong.com/unittest/junit-4-tutorial-1-basic-usage/) :)